src/Security/UserRoleAssociationModel.cs

```diff
@@ -43,18 +43,33 @@ public class UserRoleAssociationModel
         public string ExternalUserId { get; set; }
 
         /// <summary>
-        /// Gets or sets the participant type. (i.e. User or Role)
+        /// Gets or sets the participant security type. (i.e. User or Role)
```

Comment:
I know this is not you but please put the documentation period after the last paren to avoid warning.